packages/mobile/src/components/edit/PriceAndAudienceField/PriceAndAudienceScreen.tsx

```diff
@@ -47,8 +48,8 @@ export const PriceAndAudienceScreen = () => {
     'is_scheduled_release'
   )
   const [{ value: remixOf }] = useField<RemixOfField>('remix_of')
-  const [{ value: isUpload }] = useField<boolean>('is_upload')
   const [{ value: entityType }] = useField<string>('entityType')
+  const isUpload = !initialValues?.track_id
```

Comment:
@dylanjeffers I'm not sure why you changed it, but `is_upload` isn't a field?

Reply:
I see... I'll more more closely. the code as written doesn't work for distinguishing upload vs edit